coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -29,6 +32,8 @@ def __init__(self, language: str, docstyle: str, markers: (Iterable, str)):
                          or a single marker/delimiter string iterable that
                          identify a documentation comment. See ``markers``
                          property for more details on markers.
+        :param metadata: A namedtuple consisting of certain attributes that
+                         are present in the documentation comment.
```

Comment:
hmm I don't like that :D :D

it's super generic I think, "namedtuple consisting of certain attributes that are present in the documentation comment".

The question I would still have:
- what are "attributes", that are present in the documentation comment?

Also you should try to mention that it contains more descriptive properties about the layout/format of such a metadata field that appears inside a doccomment.